src/robomeshcat/human.py

```diff
@@ -4,19 +4,27 @@
 # Created on: 2022-12-6
 #     Author: Vladimir Petrik <vladimir.petrik@cvut.cz>
 #
+from __future__ import annotations
```

Comment:
Just curious, how are the annotations used? I did not see in the code

Reply:
It allows using `str | None` for type annotation instead of Optional[str]. It's supported from python 3.10, __future __ import is needed for backporting this future to python 3.7.